packages/web/src/components/DatePicker/styled.ts

```diff
@@ -136,10 +136,11 @@ export const StyledDatePicker = styled.div<{
 export const StyledHeaderFlex = styled(Flex)`
   padding: 0 5px 5px 8px;
 `;
-
-export const TodayStyledText = styled(Text)`
+//Introduced a prop that TodayStyledText will take
```

Comment:
The change looks good, but  plz delete the comment so others who stumble upon this later don't get confused